### DISCUSSION TITLE
[Bug]: I have created New directory under /var/assets/ in Pimcore server but It is not showing on Pimcore admin panel.

### DISCUSSION THREAD

**Discussion by pimcoreanand** [pimcore/pimcore#16670] [Q&A / Support] [answered]
### Pimcore version

11

### Steps to reproduce

What should I do that if created directory from Ubuntu server then same should show in Pimcore admin panel

### Actual Behavior

What should I do that if created directory from Ubuntu server then same should show in Pimcore admin panel

### Expected Behavior

What should I do that if created directory from Ubuntu server then same should show in Pimcore admin panel

**Comment by dpfaffenbauer** [comment, marked as answer]
That's not how it works. Pimcore is not scanning your folder for files but just storing them there. You have to create the files in Pimcore to be shown in the folder on the server

**Comment by danielerne** [comment]
And how should a migration work with thousands of files and multiple GBs of data? It worked on older versions.

**Reply by dpfaffenbauer** [reply]
migration in terms of copying Pimcore to another server or migrating assets that never existed in Pimcore into Pimcore?

**Reply by danielerne** [reply]
Existed in a very old version of Pimcore 6.x

**Reply by AlephLau** [reply]
In that case @danielerne  you have to move all the assets to the new server and migrate the database as well to avoid losing the reference to your images
